app/controllers/menu_controller.rb

```diff
@@ -0,0 +1,15 @@
+class MenuController < ApplicationController
+  def index
+    @menu_items = MenuItem.all
+  end
+
+  def create
```

Comment:
the create action is out of scope for this feature, please remove it

Reply:
Removed until later feature.